tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -481,6 +482,13 @@ void ordering_test_cases() {
 
         spaceship_test<std::strong_ordering>(c_mem[0], c_mem[0], c_mem[1]);
     }
+    { // thread::id
+        std::thread::id id1;
+        std::thread::id id1_equal;
+        std::thread::id id2 = std::this_thread::get_id();
+
+        spaceship_test<std::strong_ordering>(id1, id1_equal, id2);

```

Comment:
I believe there's an implementation assumption here - nothing in the Standard appears to require that `std::thread::id{}` occurs first in the unspecified total ordering. (It's a safe assumption for us, since we use `0` and store `unsigned int`.) I'll push a comment.